metrics-operator/converter/slo_converter.md

```diff
@@ -0,0 +1,387 @@
+  - criteria:
+    - "<=+10%"
+  - criteria
+    - "<600"
+  weight: 2
+  key_sli: true
+```
+
+Thirdly, a case where `pass` criteria are set, `warn` criteria are set, but pass criteria interval and warn
+criteria interval do not intercept
+
+```yaml
+objectives:
+- sli: response_time_p95
+  displayName: "Response Time P95"
+  pass:
+  - criteria:
+    - ">200"
+    - "<400"
+  warn:
+  - criteria:
+    - ">600"
+    - "<800" 
+```
+
+### Supported use-cases
+
+The basic objective with a single rule for pass or warning criteria
+
+```yaml
+objectives:
+- sli: response_time_p95
+  displayName: "Response Time P95"
+  pass:
+  - criteria:
+    - ">400"
+  warn:
+  - criteria:
+    - ">200"
+```
+
+will be converted to
+
+```yaml
+spec:
+  objectives:
+  - analysisValueTemplateRef:
+      name: response_time_p95
+      namespace: default
+    target:
+      failure:
+        lessThanOrEqual:
+          fixedValue: "200"
+      warning:
+        lessThanOrEqual:
+          fixedValue: "400"
+```
+
+The buckets for rules with single criteria element but with one or more criteria combined with logical AND
+operator (see documentation [here](https://github.com/keptn/spec/blob/master/service_level_objective.md#objectives)):
+
+1. `pass` criteria set, `warn` criteria not set
+
+```yaml
+objectives:
+- sli: response_time_p95
+  displayName: "Response Time P95"
+  pass:
+  - criteria:
+    - ">400"
+    - "<600"
+```
+
+will be converted to
+
+```yaml
+spec:
+  objectives:
+  - analysisValueTemplateRef:
+      name: response_time_p95
+      namespace: default
+    target:
+      failure:
+        notInRange:
+          highBound: "600"
+          lowBound: "400"
+```
+
+1. `pass` criteria set, `warn` criteria set, warn criteria interval is superset of pass criteria interval
+
+```yaml
+objectives:
+- sli: response_time_p95
+  displayName: "Response Time P95"
+  pass:
+  - criteria:
+    - ">400"
+    - "<600"
+  warn:
+  - criteria:
+    - ">200"
+    - "<800" 
+```
+
+will be converted to
+
+```yaml
+spec:
+  objectives:
+  - analysisValueTemplateRef:
+      name: response_time_p95
+      namespace: default
+    target:
+      failure:
+        notInRange:
+          highBound: "200"
+          lowBound: "800"
+      warning:
+        notInRange:
+          highBound: "400"
+          lowBound: "600"
+```
+
+1. `pass` criteria set, `warn` criteria set, pass criteria interval is superset of warn criteria interval
+
+```yaml
+objectives:
+- sli: response_time_p95
+  displayName: "Response Time P95"
+  pass:
+  - criteria:
+    - ">200"
+    - "<800"
+  warn:
+  - criteria:
+    - ">400"
+    - "<600" 
+```
+
+will be converted to
+
+```yaml
+spec:
+  objectives:
+  - analysisValueTemplateRef:
+      name: response_time_p95
+      namespace: default
+    target:
+      failure:
+        notInRange:
+          highBound: "200"
+          lowBound: "800"
+      warning:
+        inRange:
+          highBound: "400"
+          lowBound: "600"
+```
+
+## Example
```

Comment:
I will move this example before the limitation

Reply:
I would keep it in the end, we have partial examples with the limitations and it's quite long and complex